src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -58,7 +58,7 @@ def create_launcher(fluent_launch_mode: LaunchMode = None, **kwargs):
     DisallowedValuesError
         If an unknown Fluent launch mode is passed.
     """
-    _process_invalid_args(kwargs["dry_run"], fluent_launch_mode, kwargs)
+    _process_invalid_args(kwargs.get("dry_run"), fluent_launch_mode, kwargs)
```

Comment:
Should `_process_invalid_args()` go through similar refactoring as well? :)

Reply:
https://github.com/ansys/pyfluent/issues/2838

Reply:
Also noting this code in the `PIMLauncher` which accepts a `start_watchdog` argument:

```
        if self.argvals["start_watchdog"]:
            logger.warning(
                "'start_watchdog' argument for 'launch_fluent()' method is not supported "
                "when starting a remote Fluent PyPIM client."
            )
```

Reply:
Updated here  - https://github.com/ansys/pyfluent/pull/2842/files#diff-f3eafcfd5d76de366b9e3a94065320abf7c555ddd71c6fc2a1b6542ab8fd330b

Thank you.